### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by frezbo**
Bump deps and fix preamble checking for `Dockerfile`.

### CODE REVIEW

File: internal/output/files.go [hunk 1]
```diff
@@ -137,7 +137,10 @@ func splitIgnoringPreamble(r io.Reader) ([]string, error) {
 	scanner := bufio.NewScanner(r)
 	for scanner.Scan() {
 		line := scanner.Text()
-		if inPreamble && (strings.HasPrefix(line, "#") || strings.HasPrefix(line, "<!--") || line == "" || line == "---") { // comments, skip as it might be a preamble
+
+		// `# syntax = ...` is a special case, it's not a preamble
+		// comments, skip as it might be a preamble
+		if inPreamble && ((!strings.HasPrefix(line, "# syntax") && strings.HasPrefix(line, "#")) || strings.HasPrefix(line, "<!--") || line == "" || line == "---") {
```
Comment (utkuozdemir) [inline internal/output/files.go:143]: Maybe make this a bit more readable by extracting part of it as a variable?

**Review by utkuozdemir — APPROVED** [review]

**Review by smira — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -31,7 +31,7 @@ jobs:
     if: (!startsWith(github.head_ref, 'renovate/') && !startsWith(github.head_ref, 'dependabot/'))
     services:
       buildkitd:
-        image: moby/buildkit:v0.12.5
+        image: moby/buildkit:v0.13.0
```
Comment (smira) [inline .github/workflows/ci.yaml:34]: should we hold on this for some time? until v0.13.1 at least?

Comment (frezbo) [reply .github/workflows/ci.yaml:34]: seemed to work :sweat_smile:

Comment (smira) [reply .github/workflows/ci.yaml:34]: I have no doubt about this repository, but I'm more concerned with other repos, sensitive to reproducibility

Comment (frezbo) [reply .github/workflows/ci.yaml]: reverted

**Pull request merged**